Add command to check ocis backup consistency (#9238)
owncloud/ocis: docs/ocis/backup.md

When using an external idp/idm/nats or blobstore, its data needs to be backed up separately. Refer to your idp/idm/nats/blobstore documentation for backup details.

> ## Backup Consistency command

Reviewer: ```suggestion
## Backup Consistency Command
```

> Infinite Scale now allows checking an existing backup for consistency. Use command

Reviewer: ```suggestion
Infinite Scale now allows checking an existing backup for consistency. Use the command:
```

```bash
ocis backup consistency -p "<path-to-base-folder>"
```

> `path-to-base-folder` needs to be replaced with the path to the storage providers base path. For example: "/var/lib/ocis/storage/users" or "$HOME/storage/users".

Reviewer: `...the path to the storage providers base path.` This sentence makes the path origin not clear, you should add the relevant envvar for reference so one can have a look how it was defined.

Author: There are multiple envvars for it. Could be `BASE_DATA_BASE` or `STORAGE_USERS_OCIS_ROOT` or even `STORAGE_SYSTEM_OCIS_ROOT`. That is why put two examples instead.

Reviewer: The docs say

STORAGE_USERS_OCIS_ROOT

The directory where the filesystem storage will store blobs and metadata. If not defined, the root directory derives from $OCIS_BASE_DATA_PATH:/storage/users.

Referencing an envvar is imho much clearer as we have very good docs about...

> `path-to-base-folder` needs to be replaced with the path to the storage providers base path. For example: "/var/lib/ocis/storage/users" or "$HOME/storage/users".

Author: ```suggestion
`path-to-base-folder` needs to be replaced with the path to the storage providers base path. Should be same as the `STORAGE_USERS_OCIS_ROOT`
```

> Use the `-b s3ng` when using an external (s3) blobstore. Note: When using this flag the blobstore must be configured via envvars or yaml file the same as in ocis. Consistency checks for other blobstores than `ocis` and `s3ng` are not supported at the moment.

Reviewer: ```suggestion

Use the `-b s3ng` option when using an external (s3) blobstore. Note: When using this flag, the path to the blobstore must be configured via envvars or a yaml file to match the configuration of the original instance. Consistency checks for other blobstores than `ocis` and `s3ng` are not supported at the moment.
```
